app/react/components/datatables/TableTitle.tsx

```diff
@@ -23,15 +23,15 @@ export function TableTitle({
     <>
       <div className={clsx('toolBar flex-col', className)} id={id}>
         <div className="flex w-full items-center gap-1 p-0">
-          <h2 className="toolBarTitle m-0 text-2xl">
+          <div className="toolBarTitle">
```

Comment:
Is it possible to keep this as h2 with the same size as before? 

The h2 tag Is a bit more specific for css selectors and screen readers

Reply:
Oh yea that's a good point

Reply:
yeah, made the h2 change, so it's easier to find the header on e2e tests. 
```suggestion
          <h2 className="toolBarTitle m-0 text-base">
```

Reply:
Cool changed it to just reduce the font size to it's previous amount, but keep the h2.